pkg/dbg/dbg.go

```diff
@@ -56,6 +59,30 @@ func generateFilename(name string, dir string, suffix string) string {
 	return path.Join(dir, name+"."+time.Now().Format(fnameFmt)+suffix)
 }
 
+func changeLogLevel() {
+	var level string
+	data, err := ioutil.ReadFile(logLevelFile)
+	if err != nil {
+		level = "info"
+	} else {
+		level = string(data)
```

Comment:
Convert to lower case in case INFO or Info is given in the file

Reply:
Fixed.